soh/src/code/z_sram.c

```diff
@@ -368,6 +368,18 @@ void Sram_InitSave(FileChooseContext* fileChooseCtx) {
                 break;
         }
 
+        int startingAge = Randomizer_GetSettingValue(RSK_STARTING_AGE);
+        switch (startingAge) {
```

Comment:
This might be another thing that's up to preference, but you should be able to put `Randomizer_GetSettingValue(RSK_STARTING_AGE)` straight in the switch there right?

Reply:
Theoretically it should work the same, although later checks like whether DoT is closed in rando settings follow similar syntax, so I opted for consistency.

Reply:
Alright that's fair then, can just leave it like this.